src/rust/ensogl/lib/select/src/component.rs

```diff
@@ -0,0 +1,403 @@
+            frp.source.chosen_entry   <+ frp.chose_entry.map(|id| Some(*id));
+            frp.source.chosen_entry   <+ opt_selected_entry_chosen.gate(&any_entry_selected);
+
+
+            // === Selection Size and Position ===
+
+            target_selection_y <- frp.selected_entry.map(|id|
+                id.map_or(0.0,entry::List::position_y_of_entry)
+            );
+            target_selection_height <- frp.selected_entry.map(|id|
+                if id.is_some() {entry::HEIGHT} else {0.0}
+            );
+            eval target_selection_y      ((y) selection_y.set_target_value(*y));
+            eval target_selection_height ((h) selection_height.set_target_value(*h));
+            eval selection_y.value       ((y) model.selection.set_position_y(*y));
+            selection_size <- all_with(&frp.size,&selection_height.value,|window,height|
+                Vector2(window.x,*height)
+            );
+            eval selection_size  ((size) model.selection.shape.sprite.size.set(*size));
+
+
+            // === Scrolling ===
+
+            selection_top_after_move_up <- selected_entry_after_move_up.map(|id|
+                id.map(|id| entry::List::y_range_of_entry(id).end)
+            );
+            min_scroll_after_move_up <- selection_top_after_move_up.map(|top|
+                top.unwrap_or(MAX_SCROLL)
+            );
+            scroll_after_move_up <- min_scroll_after_move_up.map2(&frp.scroll_position,|min,current|
+                current.max(*min)
+            );
+            selection_bottom_after_move_down <- selected_entry_after_move_down.map(|id|
+                id.map(|id| entry::List::y_range_of_entry(id).start)
+            );
+            max_scroll_after_move_down <- selection_bottom_after_move_down.map2(&frp.size,
+                |id,window_size| id.map_or(MAX_SCROLL, |id| id + window_size.y)
+            );
+            scroll_after_move_down <- max_scroll_after_move_down.map2(&frp.scroll_position,
+                |max_scroll,current| current.min(*max_scroll)
+            );
+            frp.source.scroll_position <+ scroll_after_move_up;
+            frp.source.scroll_position <+ scroll_after_move_down;
+            frp.source.scroll_position <+ frp.scroll_jump;
+            eval frp.scroll_position ((scroll_y) window_y.set_target_value(*scroll_y));
+
+
+            // === Resize ===
+            frp.source.size <+ frp.resize;
+
+
+            // === Update Entries ===
+            window_info <- all_with(&window_y.value,&frp.size, |y,size|
+                Window{position_y:*y,size:*size}
+            );
+            eval window_info ((window) model.update_after_window_change(window));
+            _new_entries <- frp.set_entries.map2(&window_info, f!((entries,window)
+                model.set_entries(entries.clone_ref(),window)
+            ));
+        }
+
+        window_y.set_value(MAX_SCROLL);
+        window_y.set_target_value(MAX_SCROLL);
```

Comment:
this has very mimnor performance problems. It first sets the value, then sets the target value m, and then starts the simulator. The simulator attaches on_every_frame event, runs a simulation step, checks that actually its done, removes the on_every_frame event etc. To skip all these steps you can just do:
```rust
window_y.set_target_value(MAX_SCROLL);
window_y.skip();
```